packages/honeycomb-assets/src/server/index.js

```diff
@@ -1,51 +1,31 @@
-import Hapi from 'hapi';
-import Hoek from 'hoek';
-import inert from 'inert';
+import honeycombServer from 'honeycomb-server';
 import path from 'path';
-import logging from 'honeycomb-logging-middleware';
-import info from 'honeycomb-info-middleware';
-import health from 'honeycomb-health-middleware';
-import status from 'hapijs-status-monitor';
 import pkg from '../../package.json';
 
-const server = new Hapi.Server();
-
-server.connection({
-  port: process.env.PORT || 3001,
-});
-
-server.register([
-  { register: inert },
-  { register: logging },
-  { register: health },
-  {
-    register: info,
-    options: {
-      pkg,
-      process,
-    },
-  },
-  {
-    register: status,
-    options: {
-      title: 'Status',
-    },
-  },
-], (registerError) => {
-  Hoek.assert(!registerError, registerError);
-
-  server.route({
-    method: 'GET',
-    path: '/{param*}',
-    handler: {
-      directory: {
-        path: path.resolve(__dirname, '..', 'client'),
+honeycombServer
+  .start({
```

Comment:
i think start on the next line is more readable, because of the big config-object